core/scripts/functions/templates/oracle.toml

```diff
@@ -11,17 +11,25 @@ relay = "evm"
 pluginType = "functions"
 transmitterID = "{{node_eth_address}}"
 observationSource  = """
-	run_computation    [type="bridge" name="ea_bridge" requestData="{\\"note\\": \\"observationSource is unused but the bridge is required\\"}"]
-	run_computation
+    run_computation    [type="bridge" name="ea_bridge" requestData="{\\"note\\": \\"observationSource is unused but the bridge is required\\"}"]
+    run_computation
 """
 
 [relayConfig]
 chainID = {{chain_id}}
```

Comment:
Was this bracket deletion intentional?

Reply:
It's not a deletion IIUC, just weird coloring.